tests/framework/utils_vsock.py

```diff
@@ -160,6 +164,38 @@ def make_blob(dst_dir):
     return blob_path, blob_hash.hexdigest()
 
 
+def negative_test_host_connections(vm, uds_path, blob_path, blob_hash):
```

Comment:
I think this function would be better suited in the `test_vsock.py` file, since it's only used there and it's not really a generic negative vsock test.

Reply:
done